Description: export public symbols from ecore_fb
Author: Ross Vandegrift <ross@kallisti.us>
Forwarded: no
Last-Update: 2015-09-19
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/lib/ecore_fb/ecore_fb_private.h
+++ b/src/lib/ecore_fb/ecore_fb_private.h
@@ -33,6 +33,20 @@
 
 #include <Ecore_Fb.h>
 
+#ifdef EAPI
+# undef EAPI
+#endif
+
+#ifdef __GNUC__
+# if __GNUC__ >= 4
+#  define EAPI __attribute__ ((visibility("default")))
+# else
+#  define EAPI
+# endif
+#else
+# define EAPI
+#endif
+
 /* ecore_fb_li.c */
 struct _Ecore_Fb_Input_Device
 {
@@ -92,4 +106,7 @@
 #define TS_GET_CAL 0x8014660a
 #endif
   
+#undef EAPI
+#define EAPI
+
 #endif
